content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```diff
@@ -71,15 +71,26 @@ config. Save it as `my-scheduler.yaml`:
 
 {{< codenew file="admin/sched/my-scheduler.yaml" >}}
 
-An important thing to note here is that the name of the scheduler specified as an
-argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the optional `spec.schedulerName` on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
-
-Note also that we created a dedicated service account `my-scheduler` and bind the cluster role
+In the above manifest, we have used a [Scheduler Configuration](/docs/reference/scheduling/config/)
```

Comment:
```suggestion
In the above manifest, you use a [Scheduler Configuration](/docs/reference/scheduling/config/)
```
Under [Content best practices](https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices) in the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) suggests to use present tense and to address the reader as you not we